templates/eap-xp1-amq-s2i.json

```diff
@@ -0,0 +1,917 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+        "annotations": {
+            "iconClass": "icon-eap",
+            "tags": "eap,eap-xp,javaee,java,jboss,hidden",
+            "version": "1.0",
+            "openshift.io/display-name": "JBoss EAP XP + AMQ 7",
+            "openshift.io/provider-display-name": "Red Hat, Inc.",
+            "description": "An example JBoss Enterprise Application Platform Expansion Pack application using Red Hat JBoss AMQ. For more information about using this template, see https://github.com/jboss-container-images/jboss-eap-openshift-templates/blob/eap-xp1/README.adoc",
```

Comment:
"Example of an application using JBoss EAP XP and JBoss AMQ. For more information ..."